app/src/main/java/com/google/android/apps/flexbox/FlexItem.java

```diff
@@ -124,6 +132,8 @@ protected FlexItem(Parcel in) {
         MarginLayoutParamsCompat.setMarginStart(lp, startMargin);
         MarginLayoutParamsCompat.setMarginEnd(lp, endMargin);
         lp.bottomMargin = bottomMargin;
+        lp.minWidth = minWidth;
```

Comment:
For the width/height fields, values are stored as DP in the FlexItem class and converting to LayoutParams, convert it as pixels.

Could you follow that way for also minWidth and minHeight?

i.e. 
lp.minWidth = Util.dpToPixel(context, minWidth)
lp.minHeight = Util.dpToPixel(context, minHeight)